controllers/cps_controller_helper.go

```diff
@@ -143,28 +143,41 @@ func (r *GatekeeperReconciler) handleCPSController(ctx context.Context,
 		}
 	}()
 
+	r.subControllerWait.Done()
```

Comment:
This has to run in the goroutine or else `Done()` is called immediately after launching the goroutine.

Reply:
Oh, right--that go routine isn't blocking. 🤦‍♂️